contrib/pyzfs/libzfs_core/test/test_libzfs_core.py

```diff
@@ -1316,11 +1316,11 @@ def test_snaprange_space(self):
         lzc.lzc_snapshot([snap3])
 
         space = lzc.lzc_snaprange_space(snap1, snap2)
-        self.assertIsInstance(space, (int, long))
+        self.assertIsInstance(space, (int, int))
```

Comment:
Python 2 int is limited to 2^31-1 on 32 bit systems, in case of a value growing above that python 2 transparently switches to long and `assertIsInstance(x, (int,))` would fail. 
Thus no, we should not.

Reply:
Would something silly like

```python
# compatilbility for python 3 having no 'long' type anymore
if sys.version_info >= (3,):
    long = int
```

at the top of the file work?

Reply:
I don't think that's allowed.

Reply:
@aerusso while it's legal code it would, according to http://python3porting.com/differences.html#long (where I got what I posted from as 'this is the way') it breaks doctests.